utils/python/CIME/SystemTests/err.py

```diff
@@ -28,7 +28,7 @@ def run_phase(self):
             dout_s_root = self._case.get_value("DOUT_S_ROOT")
             rundir = self._case.get_value("RUNDIR")
             logger.info("staging files from archive %s" % dout_s_root)
-            for item in glob.glob(os.path.join(dout_s_root, "rest", "*", "*")):
+            for item in glob.glob(os.path.join(dout_s_root, "*", "hist", "*base")):
```

Comment:
Important change: I think ERR was broken before this PR, we just didn't notice it. The *base file did not exist in the "rest" subdirectory before or after this PR, so the "rest" vs. "base" comparison in ers_second_phase should not have worked.